vellum/workflows/triage_message/nodes/triage_message_node.py

```diff
@@ -82,7 +108,13 @@ class TriageMessageNode(BaseInlinePromptNode):
                 JinjaPromptBlock(
                     template="""You are triaging the latest unread message from your inbox. It was from \
 {{ contact }} and was submitted via {{ channel }}. Pick the most relevant action. Your message should \
-give the recipient confidence that you will be tending to their request and that you are working on it now.""",
+give the recipient confidence that you will be tending to their request and that you are working on it now.
+
+For job opportunities that have been forwarded to you:
+- Use job_opportunity_recruiter_response to send an enthusiastic response to the original recruiter
+- Use job_opportunity_forwarder_confirmation to confirm with the forwarder that you've responded
+- Extract the original recruiter's email address from the forwarded message content
+- Look for patterns like "From:", "Sent by:", or email signatures in forwarded messages""",
```

Comment:
Instead of putting this info in the prompt, make sure all criteria about how and when to use the function call is in the tool's docstring itself.